projects/SCEPCAL_Tully.yaml

```
@@ -0,0 +1,12 @@
+Name: SCEPCAL_Tully
+Description: Segmented Crystal Electromagnetic Precision Calorimeter (for CEPC: Circular Electron Positron Collider and possibly other future collider experiments, e.g. the FCC)
```

```suggestion
Description: "Segmented Crystal Electromagnetic Precision Calorimeter (for CEPC: Circular Electron Positron Collider and possibly other future collider experiments, e.g. the FCC)"
```

I think the colon is throwing off the yaml format